modules/logging-http-forward.adoc

```diff
@@ -15,38 +11,48 @@ Forwarding logs over HTTP is supported for both the Fluentd and Vector log colle
 .Example ClusterLogForwarder CR
 [source,yaml]
 ----
```

Comment:
The original content is correct for logging 5.8. The updated content is for logging 6.x. So, all the changes in this file, apart from stylistic changes, need to be reverted.